docs/changelog.md

```diff
@@ -12,6 +12,21 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **feature-customtabs**
+  * Now the color of the tracking protection icon adapts to color of the toolbar.
+
+* **browser-toolbar**
+  * Added a way to customize the color of the tracking protection icon via BrowserToolbar and DisplayToolbar.
+  ```kotlin
+  val toolbar = BrowserToolbar(context)
+  toolbar.trackingProtectionColor = Color.BLUE
+
+  // AND
+
+  val displayToolbar = DisplayToolbar(context, BrowserToolbar(context))
```

Comment:
DisplayToolbar is an internal class so we probably shouldn't mention it here since it's not public API.

Reply:
👍